raw_data_gen.py

```diff
@@ -0,0 +1,140 @@
+                py_path = join(kernel_path, py)
+                if not py.endswith(".py"):
+                    continue
+
+                with open(py_path, "r") as file:
+                    src = file.read()
+                    if "Original ATen:" in src:
+                        continue
+
+                kernel_name = py[:-3]
+                log_path = join(kernel_path, kernel_name + ".log")
+                pkl_path = join(kernel_path, py + ".pkl")
+                all_config_path = join(kernel_path, kernel_name + ".all_config")
+
+                # Get the kernel category
+                # Some kernels are just eliminated by the compiler
+                kernel_category = get_kernel_category(model, kernel_name, src)
+                if kernel_category is None:
+                    continue
+                
+                # Sanity check
+                if os.path.exists(log_path) and os.path.exists(pkl_path) and os.path.exists(all_config_path):
+                    seen_kernels.add(kernel_name)
+                elif not os.path.exists(log_path) or not os.path.exists(pkl_path) or not os.path.exists(all_config_path):
+                    if not kernel_name in seen_kernels:
+                        logger.warning(f"Missing {model}, {kernel_name}")
+                    continue
+                else:
+                    logger.warning(f"Incomplete {model}, {kernel_name}")
+                    continue
+
+                # Get the number of loops
+                if kernel_category is KernelCategory.REDUCTION:
+                    num_of_loops = get_number_of_loops(model, kernel_name, src)
+                else:
+                    num_of_loops = 0
+
+                kernel_counter = kernel_counter + 1
+                
+                # Map the ops to numbers
+                (reads, writes, total_bytes), nodes, node_read_writes, src_code = tuple(pickle.load(open(pkl_path, "rb")))
+                op_counts = node_read_writes.op_counts
+                op_bag = dict()
+                for op in op_counts.keys():
+                    if op in op_dict:
+                        op_bag[op_dict[op]] = op_counts[op]
+                    else:
+                        op_dict[op] = op_counter
+                        op_bag[op_counter] = op_counts[op]
+                        op_counter = op_counter + 1
+                print(model, kernel_name, kernel_category, num_of_loops)
+                print(op_bag)
+                
+                # Get the size hints from src code
+                size_hints = get_size_hints(model, kernel_name, src_code)
+                
+                # Get the stride and shape vec from flattened read_writes
+                # sort the reads/writes according to the names of buf
+                reads = sorted(reads, key=lambda x: x[0])
+                writes = sorted(writes, key=lambda x: x[0])
+                print(size_hints)
+                sizevar_allocator = SizeVarAllocator()
```

Comment:
This won't work with dynamic shapes, since you need the size hints in the original `SizeVarAllocator` instance.